packages/linux-firmware/linux-firmware.spec

```diff
@@ -48,11 +48,9 @@ Patch0010: 0010-linux-firmware-amd-ucode-Remove-amd-microcode.patch
 mkdir -p %{buildroot}/%{fwdir}
 mkdir -p %{buildroot}/%{fwdir}/updates
 
-# Here we have potential to shave off some extra space by using `install-xz` of
-# `install-zst` to compress firmware images on disk. However, that functionality
-# relies on kernels being configured with `CONFIG_FW_LOADER_COMPRESS_[ZSTD|XZ]`
-# which we currently do not have.
-make DESTDIR=%{buildroot}/ FIRMWAREDIR=%{fwdir} install
+# Use xz compression for firmware files to reduce size on disk. This relies on
+# kernel support through FW_LOADER_COMPRESS (and FW_LOADER_COMPRESS_XZ for kernels >=5.19)
+make DESTDIR=%{buildroot}/ FIRMWAREDIR=%{fwdir} install-xz
 
 
 # Remove executable bits from random firmware
```

Comment:
Do any of the compressed files have the executable bit set, or can we remove this?

Reply:
Good catch. I did a check on a vanilla test install of the compressed firmware files and it came back clean, so dropped the code removing the executable bits:

```
[fedora@devbox firmware]$ find . -type f -executable
```
came back clear.